cheatsheets/cors.md

```diff
@@ -3,6 +3,12 @@
 Testing:
 `curl --head -s 'http://example.com/api/v1/secret' -H 'Origin: http://evil.com'`
 
+Some testing conditions: 
+if site: test.example.com
+Origin: testXexample.com
+Origin: test.example.com.np
+Origin: test.xexample.com
+
```

Comment:
In [GitHub Flavored Markdown](https://github.github.com/gfm/) you have to actually leave blank lines in order to create a list.

Output of you additions in GitHub:

![image](https://user-images.githubusercontent.com/18099289/32133977-53e53824-bbe3-11e7-81d2-bbd1b5704c9d.png)